qgram.go

```diff
@@ -0,0 +1,28 @@
+package edlib
+
+import (
+	"math"
+)
+
+// QgramSimilarity compute the q-gram similarity between two strings
+// Takes two strings as parameters, a split length which defines the k-gram shingle length
+func QgramSimilarity(str1, str2 string, splitLength int) float32 {
```

Comment:
In this case, it seems to be the opposite. This function calculates a distance between two strings and not a similarity (which is a good thing to have a function for the distance and one for the similarity however).
To calculate the q-gram similarity, we should use the `matchingIndex` function on this result:
```go
matchingIndex(str1, str2, QgramDistance(str1, str2))
```

Furthermore, these two new algos should be implemented in the main function `StringsSimilarity`

Reply:
You're right, I'll make the changes and update the PR soon.